packages/jest-transform/src/ScriptTransformer.ts

```diff
@@ -399,7 +399,15 @@ export default class ScriptTransformer {
     return result;
   }
 
+  /**
+   * @deprecated use `this.shouldTransform` instead
+   */
+  // @ts-ignore
```

Comment:
why is ts-ignore needed

Reply:
since it's `private` and unused in the class` I'll clarify in the ignore comment